wasm-engines/docker/wabt.Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM jwasinger/llvm-10:1.0
 
-RUN git clone --recursive --single-branch --branch bench-times https://github.com/jwasinger/wabt.git
-RUN mkdir wabt/build && cd wabt/build && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF .. && make
+RUN git clone --recursive --single-branch --branch bench-times https://github.com/ewasm-benchmarking/wabt.git && \
+    mkdir wabt/build && cd wabt/build && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF .. && make
```

Comment:
```suggestion
    mkdir wabt/build && cd wabt/build && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF -DBUILD_TOOLS=OFF .. && make
```

Reply:
Needs `BUILD_TOOLS=ON` to build wasm-interp